backend/src/test/java/com/woowacourse/ternoko/service/ReservationServiceTest.java

```diff
@@ -152,17 +160,25 @@ void findAllByCoach() {
     @Test
     @DisplayName("면담 예약시, 당일 예약을 시도하면 에러가 발생한다.")
     void createReservationTodayException() {
-        assertThatThrownBy(() -> reservationService.create(COACH2.getId(),
-                new ReservationRequest("SUDAL", LocalDateTime.now(), FORM_ITEM_REQUESTS)))
+        // given
+        coachService.putAvailableDateTimesByCoachId(COACH4.getId(), PAST_REQUEST);
+
+        // when & then
+        assertThatThrownBy(() -> reservationService.create(COACH4.getId(),
+                new ReservationRequest("SUDAL", LocalDateTime.of(NOW, THIRD_TIME), FORM_ITEM_REQUESTS)))
                 .isInstanceOf(InvalidReservationDateException.class)
                 .hasMessage(ExceptionType.INVALID_RESERVATION_DATE.getMessage());
     }
 
     @Test
     @DisplayName("면담 예약시, 과거 기간 예약을 시도하면 에러가 발생한다.")
     void createReservationException() {
-        assertThatThrownBy(() -> reservationService.create(COACH2.getId(),
-                new ReservationRequest("SUDAL", LocalDateTime.now().minusDays(1), FORM_ITEM_REQUESTS)))
+        // given
+        coachService.putAvailableDateTimesByCoachId(COACH4.getId(), PAST_REQUEST);
```

Comment:
PAST_REQUEST가 어떤 시간인지 이름에서도 볼 수 있으면 좋을 것 같긴해요.
```suggestion

coachService.putAvailableDateTimesByCoachId(COACH4.getId(), MINUS_2_DAYS_REQUEST);
```